lib/LanguageServerCodeTransform/Model/NameImport/NameImporterResult.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Phpactor\Extension\LanguageServerCodeTransform\Model\NameImport;
+
+use Phpactor\CodeTransform\Domain\Refactor\ImportClass\NameImport;
+use Phpactor\LanguageServerProtocol\TextEdit as LspTextEdit;
+use Throwable;
+
+class NameImporterResult
```

Comment:
This VO will later be used to transport the name import result back to the completion handler.